### PR TITLE
Add extra display functions for individual polls

### PR DESCRIPTION
**Pull request by kmd-thomasBrace**
This PR adds two extra functions for displaying information about a single poll

The first function displays the total number of poll votes for an individual poll given a poll ID, if the poll does not exist, 0 is returned

The second function displays the time an individual poll was posted. The function also accepts another parameter for the date format the user wants the posted date to return. If the poll does not exist, a date with the timestamp of 0 is returned (1st Jan 1970)

This PR also updates the README with the relevant functions

### CODE REVIEW

File: wp-polls.php [hunk 1]
```diff
@@ -865,6 +879,21 @@ function get_pollvoters($display = true) {
 	}
 }
 
+### Function: Get Poll Time Based on Poll ID and Date Format
+if(!function_exists('get_polltime')) {
+	function get_polltime($poll_id, $date_format = "d/m/Y", $display = true) {
```
Comment (lesterchan) [inline wp-polls.php]: can we use single quotes? `'d/m/y'`

File: wp-polls.php [hunk 2]
```diff
@@ -865,6 +879,21 @@ function get_pollvoters($display = true) {
...
+		global $wpdb;
+		$poll_id = (int) $poll_id;
+		$timestamp = (int) $wpdb->get_var( $wpdb->prepare("SELECT pollq_timestamp FROM $wpdb->pollsq WHERE pollq_id = %d LIMIT 1", $poll_id));
+		$formatted_date = date($date_format,$timestamp);
```
Comment (lesterchan) [inline wp-polls.php]: `date( $date_format ,$timestamp );`

File: wp-polls.php (hunk 1 above)
Comment (kmd-thomasBrace) [reply wp-polls.php]: Comment Addressed!

File: wp-polls.php (hunk 2 above)
Comment (kmd-thomasBrace) [reply wp-polls.php]: Comment Addressed!

File: wp-polls.php [hunk 3]
```diff
@@ -881,11 +881,11 @@ function get_pollvoters($display = true) {
 
 ### Function: Get Poll Time Based on Poll ID and Date Format
 if(!function_exists('get_polltime')) {
-	function get_polltime($poll_id, $date_format = "d/m/Y", $display = true) {
+	function get_polltime($poll_id, $date_format = 'd/m/Y', $display = true) {
 		global $wpdb;
 		$poll_id = (int) $poll_id;
 		$timestamp = (int) $wpdb->get_var( $wpdb->prepare("SELECT pollq_timestamp FROM $wpdb->pollsq WHERE pollq_id = %d LIMIT 1", $poll_id));
-		$formatted_date = date($date_format,$timestamp);
+		$formatted_date = date( $date_format,$timestamp );
```
Comment (lesterchan) [inline wp-polls.php]: `date( $date_format, $timestamp );` the space is needed after the comma.

Comment (kmd-thomasBrace) [reply wp-polls.php]: Addressed!

**Pull request merged**